build.sh

```diff
@@ -46,6 +46,7 @@ HELP="$0 [<target> ...] [<flag> ...]
    --codecov         - Enable code coverage support by compiling with Cython linetracing
                        and profiling enabled (WARNING: Impacts performance)
    --ccache          - Use ccache to cache previous compilations
+   --nobuild         - Invoke CMake without actually building
```

Comment:
Another name might be `--configure-only`. Just a proposal, no strong feelings.

Reply:
Also, I prefer `--no-build` over `--nobuild`, if we choose to keep the current name.

Reply:
Me, too, but this was more consistent with the other option names. Regardless, I changed it to `--configure-only` now.

Reply:
https://github.com/rapidsai/cuml/pull/5235/commits/058bd31230c41b571ba6613d43254ce09eb5430a